init.sls

```diff
@@ -10,16 +10,30 @@ duplicity_ppa_repo:
 
 duplicity:
   pkg.installed:
-    {% if 'ftp://' in pillar['duplicity']['backend'] %}
-    - pkgs: [duplicity, lftp]
-    {% elif 'boto3+s3' in pillar['duplicity']['backend'] %}
-    - pkgs: [duplicity, python3-boto3]
-    {% elif 's3' in pillar['duplicity']['backend'] %}
-    - pkgs: [duplicity, python3-boto]
-    {% elif 'scp://' in pillar['duplicity']['backend'] %}
-    - pkgs: [duplicity, python3-paramiko]
+    - pkgs:
+      - duplicity
+    {% if 'ftp://' in pillar['duplicity']['backend'] or 'ftp://' in pillar['duplicity']['multi_config'] %}
+      - lftp
+    {% elif 'boto3+s3://' in pillar['duplicity']['backend'] or 'boto3+s3://' in pillar['duplicity']['multi_config'] %}
+      - python3-boto3
+    {% elif 's3' in pillar['duplicity']['backend'] or 's3' in pillar['duplicity']['multi_config'] %}
+      - python3-boto
+    {% elif 'scp://' in pillar['duplicity']['backend'] or 'scp://' in pillar['duplicity']['multi_config'] %}
+      - python3-paramiko
     {% endif %}
 
+# Insert duplicity multi configuration
+{% if 'multi://' in pillar['duplicity']['backend'] %}
+# Deploy scripts
+/etc/xdg/duplicity/multi.json:
```

Comment:
Any specific reason you chose `/etc/xdg` for placement? I can't find anything about this being some sort of default for duplicity, and `xdg` is supposed more for desktops, right?

Reply:
I thought it was a more general place for configs like `$HOME/.config` which is also defined in [the specification](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html).

But `/etc/duplicity/multi.json` might be a better place for this config? Or do you have another suggestion?

Reply:
I just think that `XDG` in general refers to desktop systems, whereas this formula is mostly used on servers.

`/etc/duplicty` sounds good!